.github/workflows/package_for_release.yml

```
@@ -1,6 +1,11 @@
 name: package for release
 on:
-    workflow_dispatch: {}
+    workflow_dispatch:
+        inputs:
+            version:
+                type: string
+                description: Release version
+                required: true
```

```suggestion
    workflow_dispatch:
        inputs:
            docker_tag:
                type: string
                description: Docker image tag prefix
                required: false
```
skip docker build if not provided